### ISSUE TITLE
Dockerfile parser on standalone

### ISSUE THREAD

**Issue by ttben** [docker-java/docker-java#742]
Hi !

I'm currently working with docker and was looking for a dockerfile parser and ended up on your repository.

I have two questions:

- don't you think it would be interesting to have the dockerfile parser (Dockerfile class and related) as a standalone artefact?
- how to extract information from the Dockerfile class? 

` Dockerfile l = new Dockerfile(new File("..."), new File("")); `
`Iterable<DockerfileStatement> statements = l.getStatements();`
`statements.forEach(d -> {
...
            });`

Basically I want to either extract enough information to convert your model into mine or, process queries such as 

> get all RUN commands
> for each command found, is there an _apt-get install_ and where? 

Thanks!
